### PR TITLE
Refactor: Remove Builder in World code

### PR DESCRIPTION
**Pull request by 0xD-Fabio**
## What does this PR change?

This PR deletes all the code from the Builder in World, which was slowing down the build.
Also, some parts have been refactored because there were some BIW utilities that still were used in other parts of the code.

...

## How to test the changes?

BIW wasn't visible/useable before this PR, so there is no difference UX side.
The test may be useful to catch unforeseen issues.

1. Go to: https://play.decentraland.zone/?renderer-branch=refactor/remove-builder-world-code-03

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Configuration/Configuration.cs [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
 namespace DCL.Configuration
 {
-    public static class BIWSettings
+    public static class AssetCatalogUrlSettings // EX: BIWSettings
     {
-        //Note: Don't use these URL directly, you need to get them in the BuilderInWorldUtils to take into account the ENV
+        //Note: Don't use these URL directly, you need to get them in the AsseteCatakigUrlUtils (before known as BuilderInWorldUtils) to take into account the ENV
```
Comment (Kinerius) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Configuration/Configuration.cs:8]: Typo

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs [hunk 2]
```diff
@@ -130,7 +126,7 @@ public void Initialize()
 
         //view.ConfigureEncapsulatedSection(ExploreSection.Backpack, DataStore.i.exploreV2.configureBackpackInFullscreenMenu);
         view.ConfigureEncapsulatedSection(ExploreSection.Map, DataStore.i.exploreV2.configureMapInFullscreenMenu);
-        view.ConfigureEncapsulatedSection(ExploreSection.Builder, DataStore.i.exploreV2.configureBuilderInFullscreenMenu);
+        // view.ConfigureEncapsulatedSection(ExploreSection.Builder, DataStore.i.exploreV2.configureBuilderInFullscreenMenu);
```
Comment (Kinerius) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs:129]: We can safely just delete this line

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Tests/ExploreV2MenuComponentControllerTests.cs [hunk 3]
```diff
@@ -347,7 +311,7 @@ public void GoToSectionCorrectly(ExploreSection section)
     [TestCase(ExploreSection.Explore)]
     [TestCase(ExploreSection.Backpack)]
     [TestCase(ExploreSection.Map)]
-    [TestCase(ExploreSection.Builder)]
+    // [TestCase(ExploreSection.Builder)]
```
Comment (Kinerius) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Tests/ExploreV2MenuComponentControllerTests.cs:314]: Delete

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/AssetCatalogUrlUtils/AssetCatalogUrlsUtils.cs [hunk 4]
```diff
@@ -1,24 +1,30 @@
...
 {
-    public static string GetUrlSceneObjectContent() { return BIWSettings.BASE_URL_SCENE_OBJECT_CONTENT.Replace("{ENV}", GetEnvBase()); }
+    public static string GetUrlSceneObjectContent() 
+    { 
+        return AssetCatalogUrlSettings.BASE_URL_SCENE_OBJECT_CONTENT.Replace("{ENV}", GetEnvBase()); 
+    }
 
     public static string GetManifestJSON(string jsonManifest) { return "{\"manifest\":" + jsonManifest + "}"; }
     
     public static string GetBuilderAPIBaseUrl()
```
Comment (Kinerius) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/AssetCatalogUrlUtils/AssetCatalogUrlsUtils.cs:12]: Is this still in use? I would just delete it if not, the same goes with `BASE_URL_BUILDER_API`

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/AssetCatalogUrlUtils/AssetCatalogUrlsUtils.cs [hunk 5]
```diff
@@ -1,24 +1,30 @@
 using DCL.Configuration;
 
-public static class BIWUrlUtils
+public static class AssetCatalogUrlsUtils
```
Comment (Kinerius) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/AssetCatalogUrlUtils/AssetCatalogUrlsUtils.cs:3]: The name is a bit unreadable, I like `AssetCatalogUtils` for clarity, but it's on you.

**Review by Kinerius — APPROVED** [review]
Really nice work! The pipeline is 35% faster with this PR.

I left some small comments.

**Review by kwspgoliasz — APPROVED** [review]
The changes appear to have been implemented correctly.

**Pull request merged**